src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/NoCodeInFileCheck.xml

```diff
@@ -5,7 +5,9 @@
              name="NoCodeInFile"
              parent="com.puppycrawl.tools.checkstyle.TreeWalker">
          <description>&lt;p&gt;
- Checks whether file contains code. Files which are considered to have no code:
+ Checks whether file contains code.
```

Comment:
You need to update template file that by this change

Reply:
Done, Updated the Check description too !!